### ISSUE TITLE
Is this a cheat that allow me to see other player's cards?

### ISSUE THREAD

**Issue by paccinolees** [Zehmosu/PokerNow#1]
Hey, not sure if ur still online but just wondering whether this is a cheat that can allow me to see other player's cards? 

If not, would it be possible to make one? I'm willing to pay for it :)

**Comment by Zehmosu** [comment]
> Hey, not sure if ur still online but just wondering whether this is a cheat that can allow me to see other player's cards?
> 
> If not, would it be possible to make one? I'm willing to pay for it :)

Nah, but (as of the time that it was uploaded) it can read the DISPLAYED cards. It's essentially a real-time webscraper using webdrivers; I haven't touched the project in a while, but it would be possible, just not without extremely heavy reverse engineering. 

I'd like to poke at it again sometime just out of pure curiosity, but I don't think I can share or release something like that without some actual serious legality issues

however, from what i remember , the project is pretty much built around the idea of trying to make my first Counterfactual Regret Minimization model, could potentially see something like that in the future, but i don't have any plans on this anytime soon, sorry sir

**Comment by kcraftcode** [comment]
@Zehmosu @paccinolees 
I can help you.

**Comment by kcraftcode** [comment]
https://crowdworks.jp/public/jobs/11628691
https://crowdworks.jp/proposals/246259862
